### PR TITLE
FIXED freeform strokes having non-manifold 'spine' faces left over

### PR DESCRIPTION
**Pull request by naam00**
(bear with me, first pull request, hope my setup is correct)
Just a tiny change: delete the intermediate face when creating a multi-step freeform brush shape.

### CODE REVIEW

File: Assets/Scripts/tools/Freeform.cs [hunk 1]
```diff
@@ -933,15 +933,18 @@ private void AddCheckpoint()
                 // Create the new 'side faces' with clockwise ordering. Each side face has 4 verts.
                 // The first face 'loops around'.
                 List<int> newFaceVertexIds = new List<int>() {
-            currentFrontFace.vertexIds[i-1],
-            currentFrontFace.vertexIds[i % vertexCount],
-            newVertices[i % vertexCount].id,
-            newVertices[i-1].id
-          };
+					currentFrontFace.vertexIds[i-1],
```
Comment (mikeage) [inline Assets/Scripts/tools/Freeform.cs:936]: We use `dotnet format` to do some code formatting, and this change wasn't so well liked :-)

Can I ask you to please undo  this part, and just keep the actual change you made on line 945?

**Review by mikeage — DISMISSED** [review]

File: Assets/Scripts/tools/Freeform.cs (hunk 1 above)
Comment (mikeage) [reply Assets/Scripts/tools/Freeform.cs:936]: that said, I'm not sure *why*; it looks like the formatting here (in our original code) was wrong! But we'll fix that later in a separate PR.

If you're not sure how to undo this change, that's cool -- just let me know, and if you want, I can push a commit to your branch that will do it.

Comment (naam00) [reply Assets/Scripts/tools/Freeform.cs]: Yes, was meant as a fix for the format, have my IDE on tabs. I think I changed it back now and used spaces everywhere.

Comment (mikeage) [reply Assets/Scripts/tools/Freeform.cs]: Thanks! We have an .editorconfig which many, but not all, programs respect; if you know if there's something we can add that your IDE would like, we'll be glad to do so

**Comment by mikeage** [comment]
There's a tool called pre-commit (pre-commit.com) which can help with formatting standardization (for example, if you click on the failed job, you can see what it changed).

If you install it locally (`pip install pre-commit` and then `pre-commit install`), it will automatically run on each Git commit you make, and if there's any formatting that needs to be done, it'll do it on the spot. It uses a local installation of `dotnet format`, so you do need to have that installed). To handle commits made before the hook was installed, you have to run it, once, manually: `pre-commit run -a`

**Review by andybak — APPROVED** [review]

**Pull request merged**